components/index/KeyFeatures.tsx

```diff
@@ -9,84 +7,34 @@ import {
   CardBody,
 } from '@patternfly/react-core';
 
-import communityIcon from '../../public/images/community.png';
-import enterpriseIcon from '../../public/images/enterprise.png';
+import keyFeaturesList from '@components/index/keyFeaturesList';
 import cogIcon from '../../public/images/cogs.png';
 
 const KeyFeatures = (
```

Comment:
With the current 8 key features, it is a bit too much horizontal scrolling before seeing the last item, is it possible to move to another row for items that don't fit the current page size?